scripts/run-pa11y-ci.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+#set -o xtrace
+
+# Run pa11y-ci shows any accessibility issues related to the UI.
+# NOTE: This script will not be useful if you are running it locally,
+# it is recommended that you run `pa11y-ci` in your command line instead.
+
+RESULT_DIRECTORY="accessibility-results"
+CURRENT_BRANCH_RESULT_FILE_NAME="current-branch"
+BASE_BRANCH_RESULT_FILE_NAME="base-branch"
+RESULT_FILE="$CURRENT_BRANCH_RESULT_FILE_NAME"
+
+
+usage() {
+  echo "Usage: $0 [OPTION...]"
+  echo ""
+  echo "-H, Display help command"
+  echo "-B, Do all commands on base branch (if this option is not given all commands are done on current branch)"
+  echo "-C, Checkouts the code from the base branch (note: the option -B is not required to do this)"
+  echo "-R, Runs pa11y-ci on branch (requires webservice to be running) and outputs results in a form that can be analysed"
+  echo "-A, Determines if current branch has more accessibility issues then base branch, requires the results from option -R to be available"
+}
+
+# Variables for determining which command to run
+RUN_ACCESSIBILITY_TEST="false"
+COMPARE_RESULTS="false"
+
+no_args="true"
+while getopts 'HBCRA' OPTION; do
+  case "$OPTION" in
+    H)
+      usage
+      exit 0
+      ;;
+    B)
+      RESULT_FILE="$BASE_BRANCH_RESULT_FILE_NAME"
+      ;;
+    C)
+      git checkout "$npm_package_config_base_branch"
+      echo "The base branch has been checked out, all other flags (if any) have been ignored"
+      exit 0
+      ;;
+    R)
+      RUN_ACCESSIBILITY_TEST="true"
+      if [ "$COMPARE_RESULTS" == "true" ]
+      then
+        echo "ERROR: You cannot use the R and A flags at the same time"
+        usage
+        exit 1
+      fi
+      ;;
+    A)
+      COMPARE_RESULTS="true"
+      if [ "$RUN_ACCESSIBILITY_TEST" == "true" ]
+      then
+        echo "ERROR: You cannot use the R and A flags at the same time"
+        usage
+        exit 1
+      fi
+      ;;
+    ?)
+      usage
+      exit 1
+      ;;
+  esac
+  no_args="false"
+done
+
+# Display help message if no arguments were given
+if [ "$no_args" == "true" ]
+then
+  usage
+  exit 1
+fi
+
+if [ "$RUN_ACCESSIBILITY_TEST" == "true" ]
+then
+  mkdir -p "$RESULT_DIRECTORY"
+  OUTPUT_FILE_PATH="${RESULT_DIRECTORY}/${RESULT_FILE}"
+
+  pa11y-ci --json > "$OUTPUT_FILE_PATH".json || true
```

Comment:
If you do my suggestion of installing pa11y-ci via package.json, then I think you'll need to change this to `npx run pa11y-ci...`.

Reply:
Fixed. I changed all instances of `pa11y-ci` to `npx pa11y-ci` as well as adding `pa11y` to `package.json`.